### PR TITLE
[DOCS-6353] Search Enterprise 3.0

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
Created branch and added to docs to homepage. new url is /search-enterprise/latest/

### CODE REVIEW

File: search-enterprise/latest/install/index.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+
+If you need access to the Elasticsearch server directly you have to perform the same operation:
+
+```bash
+ kubectl port-forward service/elasticsearch-master 9200:9200 -n alfresco
+```
+
+and then you can access the server http://localhost:9200/.
+
+More properties that can be used to configure the chart are available [here](https://github.com/Alfresco/acs-deployment/tree/master/helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/README.md){:target="_blank"}.
```
Comment (gravitonian) [inline search-enterprise/latest/install/index.md:395]: Link goes to page not found

**Review by gravitonian — CHANGES_REQUESTED** [review]
Built and browsed through all pages. Tested most links I think. Found only one broken link.

File: search-enterprise/latest/install/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/install/index.md:395]: This page will be available on release day, cheers.

**Review by gravitonian — APPROVED** [review]
Broken link will be available on Monday

**Pull request merged**